src/tests/system/tests/test_ldap.py

```diff
@@ -410,106 +361,55 @@ def test_ldap__authenticate_user_with_search_base_set(client: Client, ldap: LDAP
                 f"Setting option [ldap_group_search_base] to [{base}]",
                 f"Setting option [ldap_netgroup_search_base] to [{base}]",
             ]:
-                assert doc in str(log)
+                assert doc in str(log), f"String '{doc}' not found in logs!"
         case "ldap_group_search_base":
             for doc in [
                 "Got rootdse",
                 f"Setting option [ldap_search_base] to [{base}]",
                 f"Setting option [ldap_user_search_base] to [{base}]",
                 f"Setting option [ldap_netgroup_search_base] to [{base}]",
             ]:
-                assert doc in str(log)
+                assert doc in str(log), f"String '{doc}' not found in logs!"
         case "ldap_netgroup_search_base":
             for doc in [
                 "Got rootdse",
                 f"Setting option [ldap_search_base] to [{base}]",
                 f"Setting option [ldap_user_search_base] to [{base}]",
                 f"Setting option [ldap_group_search_base] to [{base}]",
             ]:
-                assert doc in str(log)
-
-
-@pytest.mark.topology(KnownTopology.LDAP)
-def test_ldap__lookup_user_default_naming_context_and_no_search_base(client: Client, ldap: LDAP):
```

Comment:
Why removing this test?

Reply:
At the time, I thought another covered the same scenario. I will add this one back in.